libs/schemas/src/utils.ts

```diff
@@ -38,6 +40,8 @@ export const linksSchema = {
 
 export const PG_INT = z.number().int().min(MIN_SCHEMA_INT).max(MAX_SCHEMA_INT);
 
+export const PG_ETH = z.bigint().min(MIN_SCHEMA_ETH).max(MAX_SCHEMA_ETH);
```

Comment:
If we want to support the max range of ETH values, then we should treat them as `string` or `BigNumber` in code.  For zod schemas, `string` is ideal since event schemas that extend them need to be serializable.  As discussed previously, BigInt doesn't support the required range for a u256 int.

Reply:
After discussion, if it it persists and serializes properly at runtime then no problem using bigint.